config/fund_loader_config/cof/cof_r3.py

```diff
@@ -0,0 +1,159 @@
+from config.fund_loader_config.cof.shared import fund_config
+from config.fund_loader_config.common_fund_config.fund_base_tree_paths import (
+    COF_R3W1_BASE_PATH,
+)
+
+COF_FUND_ID = fund_config["id"]
+COF_ROUND_3_WINDOW_1_ID = "e85ad42f-73f5-4e1b-a1eb-6bc5d7f3d762"
+APPLICATION_BASE_PATH = ".".join([str(COF_R3W1_BASE_PATH), str(1)])
+ASSESSMENT_BASE_PATH = ".".join([str(COF_R3W1_BASE_PATH), str(2)])
```

Comment:
Not a big deal, but my brain reads f-strings more intuitively than the `.join`

```suggestion
APPLICATION_BASE_PATH = f"{COF_R3W1_BASE_PATH}.1"
ASSESSMENT_BASE_PATH = f"{COF_R3W1_BASE_PATH}.2"
```